src/main/java/org/broadinstitute/hellbender/tools/spark/sv/CollectInsertSizeMetricsSpark.java

```diff
@@ -275,16 +281,6 @@ void writeHistogramPDF(final SAMFileHeader readsHeader, final String inputFileNa
         final File histogramPlotPDF = new File(HISTOGRAM_PLOT_FILE);
         IOUtil.assertFileIsWritable(histogramPlotPDF);
 
-            /* Temporarily commented out because Picard R script for this purpose doesn't produce subtitle based on library
```

Comment:
What needs to happen to fix this? I'm confused about what it's trying to accomplish. What does the plotter do when you're collecting at multiple levels -- produce multiple plots? Why would you manually need to add a subtitle?